package.json

```diff
@@ -20,5 +20,8 @@
     "nohoist": [
       "examples/**"
     ]
+  },
+  "dependencies": {
```

Comment:
There is no need to add `reassure` dependency in top-level package.json. It's a dummy file just to make Dependabot be able to notice all relevant `examples/*/package.json` files.

Reply:
This means I should remove the reassure dependency right?